projects/plugins/jetpack/extensions/blocks/simple-payments/edit.js

```diff
@@ -76,6 +77,24 @@ export class SimplePaymentsEdit extends Component {
 				} ),
 			} );
 		}
+
+		window.wp?.customize?.bind( 'change', setting => {
```

Comment:
Do you think it would be useful to add a check for the customizer before doing this bind?

Richard is working on a `getEditorType` function used to target every known editor type (post, site, etc. including the customizer): https://github.com/Automattic/jetpack/pull/23828

Reply:
> Richard is working on a getEditorType function used to target every known editor type (post, site, etc. including the customizer):

This is going to be handy! 🙏

Reply:
> Do you think it would be useful to add a check for the customizer before doing this bind?

I think I'm already doing that with `window.wp?.customize?`? AFAIK the `window.wp.customize` global is only defined in the customizer.

I'm happy to use the new `getEditorType` function though for consistency with the rest of the code, so I can wait for https://github.com/Automattic/jetpack/pull/23828 to land first.

Reply:
In this specific case we can go ahead without the `getEditorType` since we are forced to test for `wp.customize` anyway. 👍